lib/cts/cases/create.rb

```diff
@@ -378,6 +372,37 @@ def create_responder(responding_team)
     def get_correspondence_type_abbreviation
       @klass.type_abbreviation.to_sym.downcase
     end
+
+
+
+    def transition_to_closed_for_foi(kase)
+      kase.prepare_for_close
+      kase.update(date_responded: Date.today,
+                  info_held_status: CaseClosure::InfoHeldStatus.held,
+                  outcome_abbreviation: 'granted')
+      kase.close(CTS::dacu_manager)
+    end
+
+    def transition_to_closed_for_ico(kase)
+      kase.prepare_for_close
+      ico_decision = options.fetch(:ico_decision, Case::ICO::Base.ico_decisions.keys.sample )
+      kase.update_attributes(date_ico_decision_received: Date.today, ico_decision: ico_decision)
+      if kase.overturned?
+        uploader = S3Uploader.new(kase, CTS::dacu_manager)
+        uploader.add_file_to_case('spec/fixtures/ico_decision.png', :ico_decision)
+        # kase.attachments << FactoryBot.build(:case_ico_decision)
```

Comment:
We should be able to drop this line now.

Reply:
👍